invoice.go

```diff
@@ -669,6 +669,8 @@ type Invoice struct {
 	OnBehalfOf *Account `json:"on_behalf_of"`
 	// Whether payment was successfully collected for this invoice. An invoice can be paid (most commonly) with a charge or with credit from the customer's account balance.
 	Paid bool `json:"paid"`
+	// Returns true if the invoice was manually marked paid, returns false if the invoice hasn't been paid yet or was paid on Stripe.
+	PaidOutOfBand bool `json:"paid_out_of_band"`
```

Comment:
Could you add this to the changelog?

Reply:
I think this also got removed accidentally in #1405. Since it wasn't released, I don't think we need either one in the changelog since this just brings us back to the original state.